src/machi.proto

```diff
@@ -258,8 +272,9 @@ message Mpb_Request {
     optional Mpb_AppendChunkReq append_chunk = 112;
     optional Mpb_WriteChunkReq write_chunk = 113;
     optional Mpb_ReadChunkReq read_chunk = 114;
-    optional Mpb_ChecksumListReq checksum_list = 115;
-    optional Mpb_ListFilesReq list_files = 116;
+    optional Mpb_TrimChunkReq trim_chunk = 115;
```

Comment:
Yes, at some point in future we should freeze them but now we're now still in early stage. That's why I took readability that XXXXChunkReq had better be in same continuous sequence, than compatibility.